### PR TITLE
DOCSP-43442 - Add Python 3.13

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-ecosystem/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-43442
Staging - https://preview-mongodbmongokart.gatsbyjs.io/drivers/docsp-43442-python-version/motor/#language-compatibility

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/includes/language-compatibility-table-motor.rst
Comment (mongoKart) [inline source/includes/language-compatibility-table-motor.rst:1]: Tech reviewer: What version(s) of the driver does Motor 3.6 wrap?

**Review by mcmorisi — APPROVED** [review]
LGTM

File: source/includes/language-compatibility-table-motor.rst
Comment (blink1073) [reply source/includes/language-compatibility-table-motor.rst:1]: It wraps version 4.9.  Also, we should add MongoDB 8.0 to the list of supported Databases, and remove Database versions prior to 3.6.  This version is compatible with 3.6-8.0.

Comment (blink1073) [reply source/includes/language-compatibility-table-motor.rst:1]: Yes, they are partially compatible.

File: source/includes/mongodb-compatibility-table-motor.rst [hunk 1]
```diff
@@ -17,7 +18,22 @@
...
+   * - 3.6
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
```
Comment (blink1073) [inline source/includes/mongodb-compatibility-table-motor.rst:31]: We can remove the columns for MongoDB 2.6-3.4.

**Pull request merged**